changes/86.added

```diff
@@ -0,0 +1 @@
+Added user-guide for relationships and S3 storage backends.
```

Comment:
Please add these new docs to the table of contents defined in `mkdocs.yaml` so that the user can discover them!